index.d.ts

```diff
@@ -0,0 +1,34 @@
+import * as nock from 'nock';
+
+declare global {
+
+  export namespace Chai {
+
+    interface LanguageChains {
+      always: Assertion;
+    }
+
+    interface Assertion {
+      /**
+       * true if the spy was called at least once.
+       */
+      requested: Assertion;
+      /**
+       * @param count The number of recorded calls.
+       */
+      requestedWith(body: any): Assertion;
+      /**
+       * true if the spy was called exactly once.
+       */
+      requestedWithHeaders(headers: object): Assertion;
+      /**
+       * true if the spy was called exactly twice.
+       */
+      requestedWithHeadersMatch(headers: object): Assertion;
```

Comment:
This seems to be copy/pasted from the `sinon-chai` type definitions.

The code documentation is not correct, and the return type for these should be `Promise<Assertion>`.

Reply:
It is :). I'll clean it